### PR TITLE
Add additional antigen and sera attributes

### PR DESCRIPTION
**Pull request by drserajames**
Add antigen and sera attributes: `agLineage()`, `srLineage()`, `agReassortant()`, `srReassortant()`, `agStrings()`, `srStrings()`, `agContinent()`, `agNucleotideSequences()`, `srNucleotideSequences()`

### CODE REVIEW

File: R/map_props_points_antigens.R [hunk 1]
```diff
@@ -162,6 +174,31 @@ agSequences <- function(map, missing_value = ".") {
   map
 }
 
+#' @rdname agSequences
+#' @export
+`agNucelotideSequences<-` <- function(map, value) {
```
Comment (shwilks) [inline R/map_props_points_antigens.R:179]: typo here I think?

**Review by shwilks — CHANGES_REQUESTED** [review]
Looks great! Just missing tests for getting and setting nucleotide sequences, which would also pick up on the typo in the function name. 👍

File: R/map_props_points_antigens.R (hunk 1 above)
Comment (drserajames) [reply R/map_props_points_antigens.R:179]: I couldn't find the agSequences tests, but I see them now, so I will add them there.

**Review by shwilks — APPROVED** [review]

**Comment by drserajames** [comment]
Oh no! I forgot the documentation? Can I keep the version number the same & send another pull request?

**Pull request merged**